_datafiles/config.yaml

```diff
@@ -53,28 +53,28 @@ Server:
   OnLoginCommands:
   - 'emote @appears before you in a flash of <ansi fg="yellow-bold">⚡l<ansi fg="white-bold">i</ansi>gh<ansi fg="white-bold">tn</ansi>i<ansi fg="white-bold">n</ansi>g⚡</ansi>!'
   - print
-  - motd 
+  - motd
   - print
   - online
   - print
   - inbox check
-  - print 
+  - print
   # - Motd -
   #   Message of the day. This is displayed when the motd command is run.
-  Motd: Welcome to the Mud!
+  Motd: '{{ t "Motd" }}'
```

Comment:
This is interesting - one might expect the message of the day to be "ad hoc" created by an admin, but this changes that to a more planned and deliberate "message with translations".

I think once this PR moves through, we might consider an in-game command to translate specific msgId's interactively... this might make things like adding/translating a Motd a bit more natural than editing files.

Reply:
Yes, this PR is a preliminary implementation, and will need to be improved in the future to make it easier to translate and use.